__tests__/app.test.js

```diff
@@ -47,3 +46,59 @@ describe("returns all endpoints avaliable under '/api'", () => {
       });
   });
 });
+
+describe("returns an review that corresponds to the review_id that is passed in", () => {
+  test("returns an review object with the following properties: review_id, title, review_body, designer, review_img_url, votes, category, owner and created_at", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .then(({ body }) => {
+        expect(body.title).toEqual("Agricola");
+        expect(body.designer).toEqual("Uwe Rosenberg");
+        expect(body.owner).toEqual("mallionaire");
+        expect(body.review_img_url).toEqual(
+          "https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700"
+        );
+        expect(body.review_body).toEqual("Farmyard fun!");
+        expect(body.category).toEqual("euro game");
+        expect(body.created_at).toEqual("2021-01-18T10:00:20.514Z");
+        expect(body.votes).toEqual(1);
+      });
+  });
+  test("returns the correct review depending on the endpoint that the user inputs", () => {
+    return request(app)
+      .get("/api/reviews/5")
+      .then(({ body }) => {
+        expect(body.review_id).toEqual(5);
+      });
+  });
+  test("gives a 404 error when a valid but non-existant path is passed in", () => {
+    return request(app)
+      .get("/api/reviews/14")
```

Comment:
Should also test that /api/reviews/nonsense also errors out properly.

Reply:
PSQL should throw an error that can be handled with a PSQL error handler if the review_id isn't a number. e.g. if error === 22P02 then return appropriate error message.